x/treasury/genesis.go

```diff
@@ -1,33 +1,38 @@
 package treasury
 
 import (
+	"context"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/palomachain/paloma/x/treasury/keeper"
 	"github.com/palomachain/paloma/x/treasury/types"
 )
 
 // InitGenesis initializes the capability module's state from a provided genesis
 // state.
-func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
-	k.SetParams(ctx, genState.Params)
+func InitGenesis(ctx context.Context, k keeper.Keeper, genState types.GenesisState) {
+	Ctx := sdk.UnwrapSDKContext(ctx)
```

Comment:
```suggestion
	sdkCtx := sdk.UnwrapSDKContext(ctx)
```

go variables must start with small letter.

Reply:
ok,I'll update it